.github/workflows/link-checker.yml

```diff
@@ -19,15 +19,24 @@ jobs:
       # fetch external files
       - name: Fetching External files
         run: ./tools/prepare.sh
```

Comment:
We can delete this step if we are calling `npm run build` anyway.
https://github.com/balena-io/docs/blob/master/package.json#L14